build/project.json

```diff
@@ -6,6 +6,7 @@
       "Sake": "0.2.2",
       "xunit.runner.console": "2.1.0",
       "xunit.core": "2.1.0",
-      "NuGetPackageVerifier": "1.0.1-*"
+      "NuGetPackageVerifier": "1.0.1-*",
+      "SplitPackages": "1.0.1-*"
```

Comment:
Did you decide against adding it to Coherence? This would affect everybody's builds marginally.